README.md

```diff
@@ -1,27 +1,80 @@
 # ghshare
 
-> Encrypt data at rest using a user's public key on github.
+Encrypt data at rest using a user's public key on github.  
+
+Sometimes you want to share something encrypted and the recipient hasn't created a PGP key, but, they've uploaded a public key to GitHub. `ghshare` can fetch those keys and use them to encrypt a file.  
+
+## Installation
+
+### Download binary
+
+Download a [binary release](https://github.com/kjvalencik/ghshare/releases)  
+You will need to take the normal actions your platform requires to run a binary downloaded from the internet (macOS example: Enable macOS run unsigned binary, Make binary executable, Add binary to PATH, etc...)  
 
 ## Usage
 
 ```sh
-ghshare -h
+ghshare command [flags] [options]
 ```
 
-### GitHub Enterprise
+Available commands:  
 
-If your instance of GitHub Enterprise does not have a dedicated API host, you
-will need to suffix the host with `/api/v3`.
+### encrypt
 
-```sh
-ghshare encrypt -h https://example.com/api/v3 ...
+Basic form of encryption command:  
+
+```
+ghshare encrypt --input <plaintextFileName> --output <ciphertextFileName> --recipient <receiversGithubUsername>
+```
+
+> :information_source: Each of the receiver's GitHub public keys (available here: https://api.github.com/users/receiversGithubUsername/keys) will be used to encrypt the input file.
```

Comment:
Technically, they only encrypt the encryption key and not the file, but I don't think that's a meaningful difference for users.

Reply:
[Made the statement more generic](https://github.com/kjvalencik/ghshare/pull/13/commits/73c4aef3b3b47dc07501abdb4fa8c4c0b8006470#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R31)